gridstatus/ieso.py

```diff
@@ -41,6 +41,25 @@
 # files for these future forecasts, they are in the current day's file.
 MAXIMUM_DAYS_IN_FUTURE_FOR_ZONAL_LOAD_FORECAST = 34
 
+"""REAL TIME FUEL MIX CONSTANTS"""
+FUEL_MIX_INDEX_URL = "http://reports.ieso.ca/public/GenOutputCapability/"
+
+# Updated every hour and each file has data for one day.
+# The most recent version does not have the date in the filename.
+FUEL_MIX_TEMPLATE_URL = f"{FUEL_MIX_INDEX_URL}/PUB_GenOutputCapability_YYYYMMDD.xml"
+
+# Number of past days for which the complete generator report is available.
```

Comment:
Historical reports go back to 2015. The individual generator report goes back 90 days.